cmd/clusterctl/internal/test/fake_variable_client.go

```diff
@@ -17,9 +17,18 @@ limitations under the License.
 package test
 
 import (
+	"fmt"
+
 	"github.com/pkg/errors"
 )
 
+var (
+	// PrivateGithubDomain is the private github enterprise domain used int tests.
+	PrivateGithubDomain = "github.corp.co"
```

Comment:
Should this be a const, and if so, are we guaranteed that all private GitHub domains share this url format?